### ISSUE TITLE
Missing reference to video recordings

### ISSUE THREAD

**Issue by oaubert** [europarl/open-data-beta-testing#8]
I do not find any reference in the data/data-structures to the recordings available from https://multimedia.europarl.europa.eu/ eg a link to https://multimedia.europarl.europa.eu/en/webstreaming/plenary-session_20220215-0900-PLENARY as a resource for https://data.europarl.europa.eu/eli/dl/doc/PV-9-2022-02-15-FNL/en (and I could not access this last link to check if the video link is present inside).

I am participating to the OpenParliamentTV project, which aims at providing a multimodal interface between proceedings and video recordings, and this information is required to correctly link associated resources.

**Comment by andrea-perego** [comment]
@oaubert , indeed, the links between documents and recordings are not currently included in the data. We will take into account this issue in future releases, although we cannot at the moment give a precise indication of when it will be implemented.

About document URIs as  https://data.europarl.europa.eu/eli/dl/doc/PV-9-2022-02-15-FNL/en , these will be resolvable once the European Parliament Open Data Portal will be published.
